test.js

```diff
@@ -0,0 +1,20 @@
+const auth = require("./auth-lib");
+
+async function execute() {
+  return await auth.getAuthCode(
+    "<email>",
+    "<password>",
+    "<client-id>",
+    "<client-secret>"
```

Comment:
The inputs need to be in this order to match the expected order in the getAuthCode function

```suggestion
    "<client-id>",
    "<client-secret>",
    "<email>",
    "<password>"
```